Python/bytecodes.c

```diff
@@ -1185,23 +1299,25 @@ dummy_func(
             // Keep in sync with _common_constants in opcode.py
             switch(oparg) {
             case CONSTANT_ASSERTIONERROR:
-                value = PyExc_AssertionError;
+                value = PyStackRef_FromPyObjectNew(PyExc_AssertionError);
```

Comment:
`PyExc_AssertionError` is immortal (or the old code wouldn't have worked)
Maybe make that clear:
```suggestion
                value = PyStackRef_FromPyObjectImmortal(PyExc_AssertionError);
```
where `PyStackRef_FromPyObjectImmortal` is the same as `PyStackRef_FromPyObjectBorrow` and asserts that the object is immortal.